imp_services/displayLogs.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Created by Trevor on 7/3/2015.
+ */
+var Q = require("q");
+
+/* var a = {};
+ a["key1"] = "value1";
+ a["key2"] = "value2";
+ */
+
+var LogTypeMap = {};
+LogTypeMap[100] = {type: "Added Pile", callFunction:toStringDefault};
+LogTypeMap[200] = {type: "Added Run", callFunction:toStringDefault};
+LogTypeMap[300] = {type: "Removed Pile", callFunction:toStringDefault};
+LogTypeMap[400] = {type: "Removed Run", callFunction:toStringDefault};
+LogTypeMap[500] = {type: "Audited", callFunction:toStringDefault};
+LogTypeMap[600] = {type: "Noted", callFunction:toStringDefault};
+LogTypeMap[700] = {type: "Created New Product", callFunction:toStringDefault};
+LogTypeMap[800] = {type: "Created User", callFunction:toStringDefault};
+
+
+var stringLogs = [];
+var jsonString = '{"logs":[';
+
+function toStringDefault (jsonObj) {
+    var logObj = JSON.parse(jsonObj);
+    //console.log(jsonObj);
+    var actionPiece = JSON.parse(logObj.action).value;
+    //console.log("Action Piece " + actionPiece);
+    return logObj.username + " on " + logObj.time + " " + LogTypeMap[logObj.logType].type + " " + actionPiece;
+}
+
+
+
+module.exports =
+
+{
+    verifyKey :function (key) {
```

Comment:
As I asked before, please make this named "_verifyKey", the _ is a convention saying that the function should only be used internally by this tool and should not be called otherwise.